### PR TITLE
Tidy up some CI on GKE stuff.

### PR DESCRIPTION
**Pull request by spxtr**
I just got rid of `cmdlog`. If I need the streaming stuff again I'll think about it then.

I switched the kubernetes client and the github client to return a struct instead of an interface.

@krousey

### CODE REVIEW

File: ciongke/cmd/hook/main.go [hunk 1]
```diff
@@ -73,6 +73,22 @@ type Event struct {
...
+	ListPods(labels map[string]string) ([]kube.Pod, error)
+	DeletePod(name string) error
+
+	ListJobs(labels map[string]string) ([]kube.Job, error)
+	CreateJob(j kube.Job) (kube.Job, error)
+	DeleteJob(name string) error
+}
+
+// githubClient is satisfied by github.Client.
+type githubClient interface {
```
Comment (krousey) [inline ciongke/cmd/hook/main.go]: Where do you use these?

**Comment by krousey** [comment]
One minor question, otherwise LGTM

File: ciongke/cmd/hook/main.go (hunk 1 above)
Comment (spxtr) [inline ciongke/cmd/hook/main.go]: I use the `kubeClient` interface in `buildPR` and `deletePR` plus their tests. I use the `githubClient` interface in `validAuthor` and its tests.

Comment (spxtr) [inline ciongke/cmd/hook/main.go]: Should I move the interface definitions closer to the functions that use them? Right now I have them by the struct that includes them.

Comment (krousey) [inline ciongke/cmd/hook/main.go]: No. I had a momentary mental lapse. This is fine. :smile:

**Comment by krousey** [comment]
LGTM

**Comment by spxtr** [comment]
Thanks.

**Pull request merged**